Bottom: 462c1ad16fc010e67e047e83ca700a2840b1cc7f
Top:    451f0498bf4b773c5546df982a92c9b3c1de1e6a
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2014-08-10 22:06:56 -0400

Refresh of tyler

---

diff --git a/sound/soc/generic/simple-card.c b/sound/soc/generic/simple-card.c
index 7aeabef..b389d9c2 100644
--- a/sound/soc/generic/simple-card.c
+++ b/sound/soc/generic/simple-card.c
@@ -13,7 +13,6 @@
 #include <linux/module.h>
 #include <linux/of.h>
 #include <linux/platform_device.h>
-#include <linux/of_platform.h>
 #include <linux/string.h>
 #include <sound/simple_card.h>
 #include <sound/soc-dai.h>
@@ -157,6 +156,7 @@ asoc_simple_card_sub_parse_of(struct device_node *np,
 				     "system-clock-frequency",
 				     &dai->sysclk);
 	}
+
 	return 0;
 }
